src/components/animatedImage/index.js

```diff
@@ -43,11 +44,12 @@ class AnimatedImage extends BaseComponent {
     this.state = {opacity: new Animated.Value(0), isLoading: true};
   }
 
-  onLoad = () => {
+  onLoad = (args) => {
```

Comment:
If i'm not mistaken you should use spread operator here.. 
```suggestion
  onLoad = (...args) => {
```